File: c/4-functions/workout/fw-2.c
#include <stdio.h>
#include <math.h>
int main()
{
    int a ,b ;
    scanf("%d\n",&a);
    scanf("%d",&b);
    if(a<0||b<0)
    {
        printf("Invalid input");
    }
    else
    {
    int result = pow(a,b);
    printf("%d",result);
    }
    
}

File: java/6-functions/workout/fw-2.java
import java.util.*;

public class A{
    
    
     public static void convert(int a,int b)
    {
        if(a < 0 || b< 0)
            System.out.println("Invalid input");
        else{
        double result = Math.pow(a,b);
        System.out.println((int)result);
        }
        
    }
    
    
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int base = sc.nextInt();
        int exp = sc.nextInt();
        convert(base,exp);
       
    }
}